readme.md

```diff
@@ -1,7 +1,31 @@
 # Projects Supported by The Network Steward
 
-* **CURRENT DEADLINE**: NO round currently active
-* **CURRENT ROUND BUDGET**: -
+* **CURRENT DEADLINE**: Midnight UTC - May 22nd 2021
+* **CURRENT ROUND BUDGET**: 40mn PKT
+
+## IMPORTANT RECOMMENDATIONS
+Following a conversaion between the members of the Network Steward team, it has been decided that
+the Network Steward shall make the following **recommendations** to applicants:
+
+1. Applicants are recommended to provide in their applications, a commitment to including, in
```

Comment:
What happens if developers are compensated in PKT? I imagine it'd be booked as "equivalent $" like any other project expenditure right? If so, we should probably mention this somewhere.

Reply:
See: https://github.com/pkt-cash/ns-projects/pull/70/files#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15R15
Projects will be PKT denominated as before, no change, this is just recommendations for them to do accounting.

Reply:
Yes I get that they don't have to be sold, just thinking to make explicit mention of wage expenses. Ok to merge as is tho.